tests/benches/promote.c

```diff
@@ -0,0 +1,44 @@
+#include <assert.h>
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <yk.h>
+#include <yk_testing.h>
+
+#ifdef DO_PROMOTE
+__attribute__((yk_promote("a", "b", "x", "z")))
```

Comment:
I thought we'd got rid of this attribute?

Reply:
So did I! Investigating.

Reply:
Fixed in db05458ef20e9e61ed2603a6e1a3032ef79707ae.

I wasn't running `cargo bench` locally, but even if I had, i might have missed this! Additional fix in 307c7ca6cc7191dba3cb79529b008b65b52f3fcd.